streaming/nifi/install.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+read -p "Enter the region: " region
+export AWS_DEFAULT_REGION=$region
+
+read -p "Enter Route53 Public Hosted Zone domain (example.com): " domain
+export TF_VAR_eks_cluster_domain=$domain
+
+read -p "Enter ACM Certificate domain (*.example.com): " cert_domain
+export TF_VAR_acm_certificate_domain=$cert_domain
+
+read -p "Enter the subdomain for NiFi (nifi): " sub_domain
+export TF_VAR_nifi_sub_domain=$sub_domain
+
+read -p "Enter NiFi username for login (admin): " username
+export TF_VAR_nifi_username=$username
```

Comment:
Do we have defaults if users want to try this out without having to provide hosted zone etc.? Make these optional